pkgs/development/libraries/cpprestsdk/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub, cmake, boost165, openssl, zlib, gcc9
+}:
+
+
+
+stdenv.mkDerivation {
+  pname = "libcpprestsdk";
+  version = "2.10.18";
+  buildInputs = [
+    boost165 openssl zlib
+  ];
+  nativeBuildInputs = [
+    cmake gcc9
+  ];
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "cpprestsdk";
+    rev = "2.10.18";
+    sha256 = "07qyip57wyzpmlgyz1fri4ljmla7ar3isddvfkm8hzv6q35bx2rn";
+    fetchSubmodules = true;
+  };
+  meta = {
```

```suggestion
  version = "2.10.18";

  src = fetchFromGitHub {
    owner = "microsoft";
    repo = "cpprestsdk";
    rev = version;
    sha256 = "07qyip57wyzpmlgyz1fri4ljmla7ar3isddvfkm8hzv6q35bx2rn";
    fetchSubmodules = true;
  };

  nativeBuildInputs = [
    cmake gcc9
  ];

  buildInputs = [
    boost165 openssl zlib
  ];

  meta = {
```

Does a newer gcc version work? If not please use gcc9Stdenv. Can we use a newer boost version?